tests/unit/utils/test_tool_executor_ai.py

```
@@ -0,0 +1,217 @@
+
+
+# ---------------------------------------------------------------------------
+# Post-AI total recalculation
+# ---------------------------------------------------------------------------
+
+
+def test_fix_recomputes_totals_after_ai_changes(monkeypatch, fake_logger):
+    """Test that fix recomputes totals after AI changes."""
+
+    class _FakeTool:
+        def set_options(self, **kwargs: Any) -> None:
```

Ruff (0.15.6)

[warning] Unused method argument: `kwargs`

(ARG002)